src/ZkFold/Symbolic/Data/ByteString.hs

```diff
@@ -147,6 +147,10 @@ class BitState c n b where
     isUnset :: c n b -> Natural -> Bool b
 
 
+instance ToConstant (ByteString n (ArithmeticCircuit a)) Natural where
+    toConstant (ByteString ac) = Haskell.foldl (\y p -> toConstant p + base * y) 0 (acOutput ac)
```

Comment:
This does the wrong thing because `acOutput` contains indices of variables so it doesn't make sense to me to add them

Reply:
Also in general there cannot be a `ToConstant (smth of arithmetic circuits) Natural` instance because arithmetic circuit might have unfed inputs (this would become apparent after we merge #214).